src/OpenFOAM/meshes/meshShapes/faceData/faceData.H

```diff
@@ -118,6 +118,9 @@ public:
              return start_;
         }
 
+        //- Return the points corresponding to this face
+        inline void points(const point* meshPoints, point* facePoints) const;
```

Comment:
I created this method to get the points that are actually in the face. This is used in [here](https://github.com/Atizar/RapidCFD-dev/pull/7/files#diff-1e5365ff76d190175d70b2a0e0074827R199).